_tests/integration/fastlane_test.go

```diff
@@ -165,40 +161,48 @@ configs:
         workflow: primary
       workflows:
         deploy:
+          description: |
+            Deploys app using *Deploy to bitrise.io* step.
```

Comment:
NOTE: might i would talk a bit more about this wf: tests, build, and deploys; the test has `retry_on_failure` test repetition mode enabled... 
might be worth passing the description to tech writers too.